core/services/pipeline/orm_test.go

```diff
@@ -88,8 +74,8 @@ func Test_PipelineORM_UpdatePipelineRun(t *testing.T) {
 		err := orm.UpdatePipelineRun(db, &run, trrs.FinalResult())
 		require.NoError(t, err)
 
-		require.Equal(t, []interface{}{float64(1), nil}, run.Outputs.Val)
-		require.Equal(t, []interface{}{nil, "Random: String, foo"}, run.Errors.Val)
+		require.Equal(t, []interface{}{nil, float64(1)}, run.Outputs.Val)
```

Comment:
Why reverse it?

Reply:
hm actually I think this might have been non-deterministic and just happen to be work when reversed. I added output indexes to the tasks to make it deterministic.